src/OpenLoco/Objects/WaterObject.cpp

```diff
@@ -1,9 +1,42 @@
 #include "WaterObject.h"
 #include "../Graphics/Colour.h"
 #include "../Graphics/Gfx.h"
+#include "../Interop/Interop.hpp"
 
 namespace OpenLoco
 {
+    // 0x004C56BC
+    bool WaterObject::validate() const
+    {
+        if (cost_index > 32)
```

Comment:
I'd prefer to keep the fields that are not the same in the comparison checks on separate lines. It will make it perhaps easier to identify areas that can be refactored. For example all of these factor/index pairs should really become some sort of struct.

Reply:
makes sense, no worries then